os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -51,15 +52,30 @@ extern unsigned int __PsramStackLimit;
 #define PSRAM_HEAP_BASE ((uintptr_t)&__psram_bss_end__[0])
 #define PSRAM_HEAP_LIMIT ((uintptr_t)&__PsramStackLimit)
 void os_heap_init(void){
+	int region_idx;
```

Comment:
This should be in the same conditional, `#if CONFIG_KMM_REGION >= 2` to prevent unused variable warning.